Form-Controls/index.html

```diff
@@ -1,28 +1,129 @@
+
 <!DOCTYPE html>
 <html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
 <head>
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>My form exercise</title>
+  <title>Della Bella Form</title>
   <meta name="description" content="">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Azeret+Mono:ital,wght@0,100..900;1,100..900&display=swap" rel="stylesheet">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Azeret+Mono:ital,wght@0,100..900;1,100..900&family=Inter:ital,opsz,wght@0,14..32,100..900;1,14..32,100..900&display=swap" rel="stylesheet">
 </head>
 <body>
+      <!--nav menu-->
+  <nav class="menu"> 
+    <div>
+      <img class="logo" src="img/logo.png" alt="logo river west brand">
+    <ul class="menu-itens">
+      <li><a href="#">Home</a></li> 
+      <li><a href="#">Men</a></li> 
+      <li><a href="#">Woman</a></li> 
+      <li><a href="#">Kids</a></li> 
+    </ul>
+  </div>
+  </nav>
+  <hr class="divider"> </hr>
+   <!--end nav menu-->
+
+ <!--HEADER-->
   <header>
-    <h1>Product Pick</h1>
+    <h1>NEW FALL COLLECTION</h1>
+    <h2> LOGIN TO OUR EXCLUSIVE CLUB OFFERS</h2>
+    <form class="log-user">
```

Comment:
I think this belongs in the main form in this case, though I understand why you've done it like this.

Reply:
I was so confused when I pulled these branches. I probably committed an earlier version of my code because the double H1 tags are in my last code.